src/components/Contacts/ContactDetails/ContactDetailContext.tsx

```diff
@@ -94,6 +127,8 @@ export const ContactDetailProvider: React.FC<Props> = ({ children }) => {
         handleTabChange: handleTabChange,
         editPersonModalOpen: editPersonModalOpen,
         setEditPersonModalOpen: setEditPersonModalOpen,
```

Comment:
You can remove `setEditPersonModalOpen` from the context as we no longer use it anywhere